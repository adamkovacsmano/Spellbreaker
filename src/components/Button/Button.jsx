import React, { Component } from "react";
import styles from "./Button.module.scss";

class Button extends Component {
  state = {};
  render() {
    return (
      <button
        disabled={this.props.disabled}
        onClick={this.props.onClick}
        style={{ cursor: "pointer" }}
      >
        {this.props.name}
      </button>
    );
  }
}

export default Button;
