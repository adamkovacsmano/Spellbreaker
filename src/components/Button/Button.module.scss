button {
  background-color: rgba(255, 0, 0, 0.6);
  border: 1px solid black;
  color: black;
  padding: 10px 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 10px;
  font-family: "Fondamento", cursive;
}
