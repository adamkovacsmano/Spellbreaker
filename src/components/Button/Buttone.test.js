import React from "react";
import { shallow } from "enzyme";
import Button from "./Button";

describe("CreatePost tests", () => {
  let button;
  beforeEach(() => {
    button = shallow(<Button />);
  });

  test("should match the snapshot", () => {
    expect(button).toMatchSnapshot();
  });
});
