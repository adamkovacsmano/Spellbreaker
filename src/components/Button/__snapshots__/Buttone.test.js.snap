// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreatePost tests should match the snapshot 1`] = `
<button
  style={
    Object {
      "cursor": "pointer",
    }
  }
/>
`;
